app/bundles/ReportBundle/Security/Permissions/ReportPermissions.php

```diff
@@ -11,7 +11,7 @@ public function __construct($params)
     {
         parent::__construct($params);
         $this->addExtendedPermissions('reports');
-        $this->addCustomPermission('export', ['enable' => 1024]);
+        $this->addCustomPermission('export', ['enable' => 1024, 'notAnonymize' => 2]);
```

Comment:
Can we define constants for these values: 1024, 2?

Reply:
I followed the convention that is already in effect, this is how all permissions are granted in Mautic